yt/python/yt/cli/strawberry_parser.py

```diff
@@ -8,6 +8,17 @@
 import os
 
 
+def _normalize_proxy(proxy):
+    for prefix in ["http://", "https://"]:
+        if proxy.startswith(prefix):
+            proxy = proxy[len(prefix):]
+
+    if ":" in proxy:
```

Comment:
Not sure that truncating the port is a good idea, at least when it is not a "well known" port (80/443)